health/health.d/megacli.conf

```diff
@@ -54,7 +60,10 @@ component: RAID
     every: 10s
      warn: $this <= (($status >= $WARNING)  ? (85) : (80))
      crit: $this <= (($status == $CRITICAL)  ? (50) : (40))
-     info: average battery backup unit (BBU) relative state of charge over the last 10 seconds
+     info: Average battery backup unit (BBU) relative state of charge over the last 10 seconds. \
```

Comment:
Instead to use `battery backup unit`,  I suggest we use `Battery Backup Unity`, it makes more sense considering the (BBU) that we are using to identify it.

Reply:
I think it is ok to keep it lowercase (it is lowercase in "MegaRAID SAS User's Guide")

Reply:
All right, so LGTM!